src/test/java/com/hatchways/assessment/api/ApplicationTests.java

```diff
@@ -0,0 +1,162 @@
+package com.hatchways.assessment.api;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.junit.jupiter.api.Test;
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.HashSet;
+
+class ApplicationTests {
+
+	/// helper function to get the response code of the API endpoint
```

Comment:
You should use JavaDoc pattern here instead of a regular comment. Your Java IDE will produce JavaDocs automatically.
It would look like this:

```
/**
   * Helper function to get the response code of the API endpoint
   *
   * @param String url
   * @return int Returns the
   * @throws IOException
   */
```

Reply:
Also check out best practices for testing Spring APIs - https://spring.io/guides/gs/testing-web/